pwiz/data/vendor_readers/Waters/Reader_Waters_Test.data/ATEHLSTLSEK_profile-centroid.mzML

```diff
@@ -102,7 +102,7 @@
         <cvParam cvRef="MS" accession="MS:1000127" name="centroid spectrum" value=""/>
         <cvParam cvRef="MS" accession="MS:1000504" name="base peak m/z" value="341.017578"/>
         <cvParam cvRef="MS" accession="MS:1000505" name="base peak intensity" value="2.36912e05"/>
-        <cvParam cvRef="MS" accession="MS:1000285" name="total ion current" value="1.615975e06"/>
+        <cvParam cvRef="MS" accession="MS:1000285" name="total ion current" value="1.61599e06"/>
         <scanList count="1">
```

Comment:
OK, so I hadn't realized that the SDK update changes the peak values for non-ddaProcessing data. I thought it was only changing the TIC metadata. What is the reason for the peaks changing?

Reply:
Ah ok I think we discussed this when I updated the test results - my belief is it's probably down to a different version of the C++ runtime, giving slightly different floating point calculations.

This was the thing where it now passes exactly on my machine, and your machine, but was different on the build server.

Reply:
But you had to regenerate the test files to get them to pass, right? That should mean the change comes from the SDK changes, not the CRT (I generated the old test files on my own PC, not on a TeamCity AWS agent).

Reply:
This conversation is anchored right above one of the base64 arrays that changed (I should have anchored it below, sorry). It's the `ATEHLSTLSEK_profile-centroid.mzML` file. It seems to be the only one that changed. Did something in the MassLynx SDK peak picker change?

Reply:
Hi Matt, just to let you know I'm still investigating this. Looking at the data that's changed, the m/z and intensities are similar, but with slight differences, and there's one spectrum with a couple of missing (fairly low intensity) points.